test/unit/org/apache/cassandra/repair/FailingRepairFuzzTest.java

```diff
@@ -114,6 +114,7 @@ public void failingRepair()
                 }
 
                 cluster.processAll();
+                Assertions.assertThat(repair.state.isComplete()).describedAs("Repair job did not complete, and no work is pending...").isTrue();
```

Comment:
added to produce a more useful error msg when repairs are not complete... These classes all assume that nothing left to process means the repair is complete, but it could also mean a bug in repair that will never get recovered... in that case the repair is not complete and the error message is hard to understand... so this new error message is trying to express this case